Automattic/wp-calypso
# Store: Add user facing fields for Google Analytics for eCommerce

...to site settings when WooCommerce is active on the site

See #17105 

See also https://github.com/Automattic/jetpack/pull/7817

To test:
```
* You'll need three sites - a WPCOM site with a Business plan, a site with Jetpack 5.3 and a Professional plan and a site with Jetpack 5.4-beta3 or newer and a Professional plan

* First, start with the WPCOM site with a Business Plan (but not an Atomic site)
* Navigate to http://calypso.localhost:3000/settings/traffic/DOMAIN
* Scroll down to Google Analytics
* Make sure you can edit the tracking ID (the UA... number)
* Make sure if you View Source for that site, you see the updated tracking ID

* Repeat for the Jetpack site with the running Jetpack 5.3 with a Professional Plan
* Navigate to http://calypso.localhost:3000/settings/traffic/DOMAIN
* Scroll down to Google Analytics
* Make sure you can edit the tracking ID (the UA... number)
* Make sure if you View Source for that site, you see the updated tracking ID

* Lastly, repeat for the Jetpack site with the running Jetpack 5.4-beta3 or newer with a Professional Plan
* Start with WooCommerce deactivated
* Navigate to http://calypso.localhost:3000/settings/traffic/DOMAIN
* Scroll down to Google Analytics
* Make sure you can edit the tracking ID (the UA... number)
* Make sure if you View Source for that site, you see the updated tracking ID
* Make sure you can toggle Anonymize IP on/off
* Make sure if you View Source for that site, you see the anonymize flag when enabled
* Make sure you DO NOT see the Basic Store Analytics settings

* Activate WooCommerce on that site
* Navigate to http://calypso.localhost:3000/settings/traffic/DOMAIN
* Scroll down to Google Analytics
* Make sure you DO see the Basic Store Analytics settings
* Enable tracking of Purchase Transactions
* In another tab, navigate to your Google Analytics Dashboard
* In your Google Analytics Dashboard, go to Admin for your site (it looks like a gear in the lower left corner)
* Under View select Ecommerce Settings
* Enable Ecommerce if you haven't already. DO NOT ENABLE ENHANCED ECOMMERCE AT THIS TIME.
* Open an incognito window to your site and complete a purchase
* Toggle back to the Google Analytics dashboard and go to Conversions and then Ecommerce
* Set the date range to today if needed
* Make sure you see the sale and that the details are correct

* Go back to the tab with http://calypso.localhost:3000/settings/traffic/DOMAIN
* Enable tracking Add to Cart Events
* On the incognito window, visit a few product pages and add some products to your cart.
* Go back to the tab with your Google Analytics Dashboard
* In your Google Analytics Dashboard, go to Real-time and then Events
* Make sure you see the addition of the products and that the details are correct
```
